shared/src/types/admin/OperationalExpense.ts

```diff
@@ -4,5 +4,5 @@ export type OperationalExpense = {
 	name: string;
 	type: string;
 	created: Date;
-	amount_chf: number;
+	amount_chf: number; // TODO: change to amount and add currency field?
```

Comment:
For the stats, we want to have everything in CHF. Otherwise, you need to know the exchange rate at the time of the expense. In the end, we operate in CHF.

If we want to have additional information, I would suggest adding, an amount and currency optionally to the `amount_chf`. That's also how stripe does it.